search-commons/src/test/java/no/unit/nva/search/ExportSearchResourcesTest.java

```diff
@@ -27,7 +27,8 @@ void shouldAllowCreationOfCsv() throws IOException {
         var searchResponse = SearchResponseDto.fromSearchResponse(getSearchResponseFromJson(json), randomUri());
 
         var value = ExportSearchResources.exportSearchResults(searchResponse);
-        var actual = CsvUtil.toExportCsv(value);
+        // skipping UTF-8 BOM for some reason is needed
+        var actual = CsvUtil.toExportCsv(value.substring(1));
```

Comment:
Can you extract this to a method called `removeUtfBom`to make this very clear that this is happening. Were we not in complete control, I'd even suggest ensuring that the BOM was a BOM, but we are, so don't

Reply:
Fixed